src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -1365,3 +1365,57 @@ def _compute_rp_violations_numba(nb_rp_violations, spike_trains, spike_clusters,
             spike_train = spike_trains[spike_clusters == i]
             n_v = _compute_nb_violations_numba(spike_train, t_r)
             nb_rp_violations[i] += n_v
+
+
+def compute_SD_test(
```

Comment:
```suggestion
def compute_std_test(
```
I'm not super happy with the name. What abut `sd_ratio`?

Reply:
Yeah I'm fine with that name!

`sd_ratio` or `std_ratio`?